src/slider/tp-slider.ts

```diff
@@ -32,6 +35,22 @@ export class TPSliderElement extends HTMLElement {
 		this.autoSlide();
 		this.setAttribute( 'initialized', 'yes' );
 
+		// Settings.
+		const responsiveData: string = this.getAttribute( 'responsive' ) || '';
+		this.responsiveSettings = responsiveData ? JSON.parse( responsiveData ) : [];
+		this.defaultSettings = {};
+		this.settingKeys = [
```

Comment:
@imranhsayed Lets move this out of the constructor and directly into the props above